app/domain/errors.rb

```diff
@@ -273,6 +273,18 @@ module AuthnK8s
           msg:  "Invalid Kubernetes host id: {0}. Must end with <namespace>/<resource_type>/<resource_id>",
           code: "CONJ00048E"
         )
+
+        CSRMissingCNEntry = ::Util::TrackableErrorClass.new(
+            msg:  "CSR [subject: {0}, spiffe_id: {1} must have a CN entry."+
+                " Account: {2}, Service Name: {3}",
+            code: "CONJ00058E"
+        )
+
+        CertMissingCNEntry = ::Util::TrackableErrorClass.new(
+            msg:  "Cert [subject: {0}, san: {1}, san_uri: {2}] must have a CN entry."+
+                " Account: {3}, Service Name: {4}",
```

Comment:
same here - do not print the account and service name. 

change to `Cert  with subject {0}, san {1} & san_uri {2}] must have a CN entry`